razzle.config.js

```diff
@@ -10,27 +10,46 @@ module.exports = {
       */
       appConfig.module.rules.shift();
 
-      // Setup bundle analyser
-      if (target === 'web' && !process.env.CI) {
-        const { BundleAnalyzerPlugin } = require('webpack-bundle-analyzer'); // eslint-disable-line import/no-extraneous-dependencies, global-require
-        appConfig.plugins.push(
-          new BundleAnalyzerPlugin({
-            analyzerMode: 'static',
-            defaultSizes: 'gzip',
-            generateStatsFile: true,
-            openAnalyzer: false,
-            reportFilename: '../../reports/webpackBundleReport.html',
-            statsFilename: '../../reports/webpackBundleReport.json',
-          }),
-        );
+      if (target === 'web') {
+        // setup bundle splitting
+        appConfig.output.filename = 'static/js/[name].[hash:8].js';
+        appConfig.optimization = {
+          splitChunks: {
+            chunks: 'initial',
+            automaticNameDelimiter: '-',
+            minSize: 184320, // 180kb
```

Comment:
Could you please share why these particular values were decided for min & max size? It would be good to add reasoning here for future reference. 

Likewise it would be good to note reasoning for the values used for `maxAssetSize` & `maxEntrypointSize` in the Razzle config.

Reply:
There was no science behind the values I used, they are fairly arbitrary :) would be good to do some analysis for an optimal bundle size over HTTP/2.

Reply:
Fair enough! I'd be happy to keep the values as they are in this PR and open up another issue to investigate optimal bundle sizes.